docs/intro.rst

```diff
@@ -62,7 +56,13 @@ Initialization of CritiqueBrainz database is also required::
 
 Then you can start all the services::
 
-   $ docker-compose -f docker/docker-compose.dev.yml up -d
+    $ docker-compose -f docker/docker-compose.dev.yml up -d
+
+.. seealso:: An alternative way for setting up the MusicBrainz database is to download the dumps manually and then do the import. Data dumps provided from https://musicbrainz.org can be downloaded from https://musicbrainz.org/doc/MusicBrainz_Database/Download. Then setup the MusicBrainz database using::
+
+   $ docker-compose -f docker/docker-compose.dev.yml run -v $DUMPS_DIR:/home/musicbrainz/dumps -v $PWD/data/mbdata:/var/lib/postgresql/data/pgdata musicbrainz_db
```

Comment:
It's a good idea to split long lines of commands into multiple using `\`. That way all of it is going to be visible without having to scroll.